### ISSUE TITLE
Feature idea: Mass rename HD prefix into suffix

### ISSUE THREAD

**Issue by Zexaron** [PredatH0r/ChanSort#116]
Hello

There could be a mass-rename feature which would move the HD prefix into a suffix at the end of the name.

That's been and still currently is how channels are named by default on Telemach Slovenia, with the HD prefix.

United Group serves across the balkans and the same brand name is used there, it may not be the same in other neighbouring countries, I'm not sure if the infrastructure is tightly interconnected or not.
The reason why it may not be interconnected is because this was a long term consumption of smaller operators slowly merging into this, with many name changes, under the hood there still were recently the same hostnames that were used more than 15 years ago.

With many more channels becoming HD I'm moving all the standard national ones to the beginning of my list including others, I thought this would be nice to have next time I have to redo the channel list and sort.

I provided the fresh .scm from Samsung D-Series UE40D6750 - this locale version may be also useful for other things if nobody else provided one.

[channel_list_UE40D6750_1101.zip](https://github.com/PredatH0r/ChanSort/files/2397458/channel_list_UE40D6750_1101.zip)

Thanks

PS: Future extension of this idea could morph into the "Quality Designator Manager" with more flexibility, it would detect the HD moniker, move it around, select which channels or everyone, and also use custom moniker instead while doing the rename, might be handy when there will be more types of resolutions in the future on one channel list.

**Comment by Zexaron** [comment]
Yeah, I remember now there's a bug with this network, the HIDE property, it's the other way around, all channels are marked as hidden while they're not, marking them as unhidden would make them hidden on the TV, I think it was the same across all the Samsung TVs, including newer H models.

**Comment by brainhex** [comment]
Same with my D5500 all channels marked as hidden but in fact some channels are visible in tv channel list and others (scrambled) not. My problem is that i made a channel search and all the crypt channels are hidden in list and i can make them visible in channel list.
